images/manageiq-orchestrator/container-assets/entrypoint

```diff
@@ -16,6 +16,54 @@ function check_svc_status() {
   echo "${SVC_NAME}:${SVC_PORT} - accepting connections"
 }
 
+function check_deployment_status() {
+  echo "== Checking deployment status =="
+
+  cd ${APP_ROOT}
+  while true; do
+    bin/rake evm:deployment_status
+    case $? in
+      3) # new_deployment
+        echo "== Starting New Deployment =="
+
+        pushd ${APP_ROOT}
+          set -e
+          REGION=${DATABASE_REGION} bin/rake db:migrate
+          REGION=${DATABASE_REGION} bin/rake db:seed
+          create_initial_group
+        popd
+
+        break
+      ;;
+      4) # new_replica
+        echo "New replica is not supported, exiting.."
+        return 1
+      ;;
+      5) # redeployment
+        echo "== Starting Re-deployment =="
+
+        break
+      ;;
+      6) # upgrade
+        echo "== Starting Upgrade =="
+        pushd ${APP_ROOT}
+          bin/rake db:migrate
+          [ "$?" -ne "0" ] && echo "ERROR: Failed to migrate database" && return 1
+        popd
+
+        break
+      ;;
+      7)
+        echo "Database has not been created yet, retrying..."
+        sleep 5
```

Comment:
Only problem here is that nothing is going to recreate the database.  But for now, this is better than straight blowing up.

Reply:
Yeah exactly figured at least this would show in the log and not have the pod constantly failing and restarting